LenovoLegionToolkit.WPF/App.xaml.cs

```diff
@@ -557,4 +577,18 @@ private static void InitMacroController()
         var controller = IoCContainer.Resolve<MacroController>();
         controller.Start();
     }
+
+    private void InitIPCServer()
```

Comment:
I think the IPC server should be optional - make a setting for it.  You can place the toggle under `Integrations`.

Reply:
I see, will do that later.